docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '3.8'
+
+services:
+  node-express:
+    image: rogerkondrat/tweeter
+    container_name: tweeter
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    ports:
+      - "5000:5000"
+      - "3000:3000"
```

Comment:
Why port 3000?

Reply:
Quoted from above: Like all things weird I had a problem once and it was a while ago and it fixed it. Sometimes when I copy and paste across most of my existing processes I forget to delete something. Thanks for pointing this out.

Also I just wrote this quickly. I never thought it would go to production as is, so no judging please ;)
Consider this just a quick and dirty draft that I put out there and I'm happy to update it and make it final after I hear there is appetite for this kind of change to our way of doing things.

Reply:
No problem! That's why we do code reviews ;)